### PR TITLE
Scanner descriptions

### PR DESCRIPTION
**Pull request by Tabrarian**
Adding the scanner descriptions doc Henry authored and updating structure, adding link to new content.

### CODE REVIEW

**Comment by Tabrarian** [comment]
This should be good to go; conflicts should resolve without any difficulty since the changes affect different parts of the page.

File: _docs/quickstarts/scanner/descriptions.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+title: Scanner Descriptions and Dependencies
+order: 102
+---
+
+# {{ page.title }}
+
+This page describes the Forseti scanners that are available, how they work, and
+why they're important. You can configure Scanner to execute multiple scanners in
+the same run. Learn about [configuring
+Scanner]({% link _docs/howto/configure/configuring-forseti#configuring-scanner %}).
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: To do this properly (and not break liquid templates you'll need to add the heading level link outside the {}, e.g.:

`({% link _docs/howto/configure/configuring-forseti.md %}#configuring-scanner)`

File: _docs/quickstarts/scanner/descriptions.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+The BigQuery scanner supports a blacklist mode, to ensure unauthorized users
+don't gain access to your datasets.
+
+For examples of how to define scanner rules for your BigQuery datasets, see the
+[bigquery_rules.yaml](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/bigquery_rules.yaml)
+rule file.
+
+The bigquery scanner depends on the following Forseti Inventory pipelines:
+
+-   `load_bigquery_datasets_pipeline`
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: The standard so far is once space between the ordered list and the list item itself, e.g.

`- `load_bigquery_datasets_pipeline``

Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: If it's not rendering correctly on the site you can indent the bulleted list by two spaces until it does.

File: _docs/quickstarts/scanner/descriptions.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
+---
+title: Scanner Descriptions and Dependencies
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: nit: This title is really long too, can we shorten it?

**Comment by Tabrarian** [comment]
I think I got everything updated; apologies on the weird bulleted list spacing, I used an auto-format and missed that it had done that.

File: _docs/quickstarts/scanner/descriptions.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+- `load_group_members_pipeline`
+
+## IAM scanner
+
+Cloud Identity and Access Management (Cloud IAM) policies directly grant access
+on Google Cloud Platform. To ensure only authorized members and permissions are
+granted in Cloud IAM policies, IAM scanner supports the following:
+
+- Whitelist, blacklist, and required modes;
+- Define whether the scope of the rule inherits from parents or just self;
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: nit is the `;` intended?

File: _docs/quickstarts/scanner/descriptions.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+title: Scanner Specifications
+order: 102
+---
+
+# {{ page.title }}
+
+This page describes the Forseti scanners that are available, how they work, and
+why they're important. You can configure Scanner to execute multiple scanners in
+the same run. Learn about [configuring
+Scanner]({% link _docs/howto/configure/configuring-forseti %}#configuring-scanner).
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: The link will need to end in .md

Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: I see this being the case in a couple places.

Comment (Tabrarian) [inline _docs/quickstarts/scanner/descriptions.md]: Got it, thanks!

File: _docs/quickstarts/scanner/descriptions.md (hunk 4 above)
Comment (Tabrarian) [inline _docs/quickstarts/scanner/descriptions.md]: Yes, although it's likely an outmoded style. I'll change it.

**Pull request merged**